yt_dlp/extractor/youtube.py

```diff
@@ -6307,6 +6317,33 @@ def _real_extract(self, url):
             ie=YoutubeTabIE, video_id=playlist_id)
 
 
+class YoutubeShortsAudioPivotIE(InfoExtractor):
+    IE_DESC = 'YouTube Shorts audio pivot; "ytshortsap:" prefix'
+    IE_NAME = 'youtube:shorts:audiopivot'
+    _VALID_URL = f'(?x)^ytshortsap:{YoutubeIE._VALID_URL[5:]}'  # TODO: might be better to just restrict this to id and /shorts/
```

Comment:
I would have restricted it to just the id, but if you want to support URL, might as well support all patterns